### PR TITLE
Feature extra testing

### PR DESCRIPTION
**Pull request by w531t4**

### CODE REVIEW

File: trivia.py [hunk 1]
```diff
@@ -35,10 +35,10 @@ def assemble_db(self, removeDisqualifyingCategories=True):
         """Consolidate JSON-oriented Quester/Answer File into a Dictionary"""
         # rather than read the file multiple times, we convert to a dictionary first
         db = {}
-        self.logger.debug("type(self.raw)=" + str(type(self.raw)))
+        #self.logger.debug("type(self.raw)=" + str(type(self.raw)))
```
Comment (MarkhamShofner) [inline trivia.py]: why are these commented out? seem like they are useful

File: main.py [hunk 2]
```diff
@@ -2,25 +2,67 @@
...
+        except:
+            players = None
+        try:
+            options = json.loads(target_scenario)['options']
+        except:
+            options = None
+
+        game_args['predetermined_spins'] = spins
+        game_args['predetermined_players'] = players
+        game_args['predetermined_startingplayer'] = options['setStartingPlayer'] if "setStartingPlayer" in options.keys() else None
```
Comment (MarkhamShofner) [inline main.py]: this line is throwing `AttributeError: 'NoneType' object has no attribute 'keys'` on me

File: trivia.py (hunk 1 above)
Comment (w531t4) [reply trivia.py]: i put them back in - i originally removed them simply because they were so noisy

File: main.py (hunk 2 above)
Comment (w531t4) [reply main.py]: resolved w/ a conditional

**Pull request merged**